eng/vmr-sync.sh

```diff
@@ -20,26 +20,26 @@
 ###   Synchronize the VMR to a specific commit of dotnet/runtime using custom fork:
 ###     ./vmr-sync.sh \
 ###        --repository runtime:e7e71da303af8dc97df99b098f21f526398c3943 \
-###        --remote runtime:https://github.com/yourfork/runtime \
-###        --tmp-dir "$HOME/repos/tmp"\
+###        --remote runtime:https://github.com/yourfork/runtime          \
+###        --tmp-dir "$HOME/repos/tmp"
 ###
 ### Options:
 ###   -t, --tmp, --tmp-dir PATH
 ###       Required. Path to the temporary folder where repositories will be cloned
-###   --repository name:GIT_REF
+###   -r name:GIT_REF, --repository name:GIT_REF
```

Comment:
The syntax here is not consistent with `tmp`.  With `tmp` the value is only mentioned once after all the option permutations.

Reply:
Good point